src/bt-proto.c

```diff
@@ -436,3 +436,23 @@ append_bt_uuid_t(struct pdu* pdu, const bt_uuid_t* uuid)
 
   return append_to_pdu(pdu, "m", uuid->uu, sizeof(uuid->uu));
 }
+
+long
+append_bt_ifname(struct pdu* pdu, const char* ifname)
```

Comment:
This function belongs into 'bt-pan-io.c'. 'bt-proto.c' only contains code that is relevant to multiple modules.